src/imgui/ImGuiSymbols.cc

```diff
@@ -158,11 +204,32 @@ static void drawTable(ImGuiManager& manager, const SymbolManager& symbolManager,
 
 			if (ImGui::TableNextColumn()) { // name
 				im::ScopedFont sf(manager.fontMono);
-				ImGui::TextUnformatted(sym.name(symbolManager));
+				im::StyleColor(sym.segment(symbolManager) && *sym.segment(symbolManager) > 0, ImGuiCol_Text, getColor(imColor::YELLOW), [&]{
+					ImGui::TextUnformatted(sym.name(symbolManager));
+				});
+				auto symNameMenu = tmpStrCat("symbol-manager##", sym.name(symbolManager)).data();
+				if (ImGui::IsItemClicked(ImGuiMouseButton_Right)) {
+					ImGui::OpenPopup(symNameMenu);
+				}
+				im::Popup(symNameMenu, [&]{ drawContext(motherBoard, sym); });
 			}
 			if (ImGui::TableNextColumn()) { // value
 				im::ScopedFont sf(manager.fontMono);
-				ImGui::StrCat(hex_string<4>(sym.value(symbolManager)));
+				ImGui::TextUnformatted(tmpStrCat(hex_string<4>(sym.value(symbolManager))).c_str());
+			}
+			if (ImGui::TableNextColumn()) { // slot
+				im::ScopedFont sf(manager.fontMono);
+				if (sym.slot(symbolManager) && sym.subslot(symbolManager)) {
+					ImGui::TextUnformatted(tmpStrCat(narrow<int>(*sym.slot(symbolManager)), "-", narrow<int>(*sym.subslot(symbolManager))));
+				} else if (sym.slot(symbolManager)) {
+					ImGui::TextUnformatted(tmpStrCat(*sym.slot(symbolManager)));
+				} else {
+					ImGui::TextUnformatted("-");
+				}
```

Comment:
This is the 2nd place where a combine primary+secondary slot field would be a bit more work. And here it would be useful to test whether the primary slot is actually expanded in the current MSX machine.